Sprint-1/implement/max.test.js

```diff
@@ -10,34 +10,47 @@ You should implement this function in max.js, and add tests for it in this file.
 We have set things up already so that this file can see your function from the other file.
 */
 
-const findMax = require("./max.js");
+const findMax = require("./max");
 
 // Given an empty array
 // When passed to the max function
 // Then it should return -Infinity
-// Delete this test.todo and replace it with a test.
-test.todo("given an empty array, returns -Infinity");
-
+test("Given an empty array, it should return -Infinity", () => {
+  expect(findMax([])).toBe(-Infinity);
+});
 // Given an array with one number
 // When passed to the max function
 // Then it should return that number
-
+test("Given an array with one number, it should return that number", () => {
+  expect(findMax([10])).toBe(10);
+});
 // Given an array with both positive and negative numbers
 // When passed to the max function
 // Then it should return the largest number overall
-
+test("Given an array with both positive and negative numbers, it should return the largest number overall", () => {
+  expect(findMax([30, 50, 10, 40])).toBe(50);
+});
 // Given an array with just negative numbers
 // When passed to the max function
 // Then it should return the closest one to zero
-
+test("Given an array with just negative numbers, it should return the closest one to zero", () => {
+  expect(findMax([-30, -50, -10, -40])).toBe(-10);
+});
 // Given an array with decimal numbers
 // When passed to the max function
 // Then it should return the largest decimal number
-
+test("Given an array with decimal numbers, it should return the largest decimal number", () => {
+  expect(findMax([30.5, 50.1, 10.2, 40.3])).toBe(50.1);
+});
 // Given an array with non-number values
 // When passed to the max function
 // Then it should return the max and ignore non-numeric values
-
+test("Given an array with non-number values, it should return the max and ignore non-numeric values", () => {
+  expect(findMax(["hey", 10, "hi", 60, 10])).toBe(60);
+});
 // Given an array with only non-number values
 // When passed to the max function
 // Then it should return the least surprising value given how it behaves for all other inputs
+test("Given an array with only non-number values, it should return -Infinity", () => {
+  expect(findMax(["hey", "hi", "hello"])).toBe(-Infinity);
```

Comment:
Could also consider checking other non-number values such as `true`, `null`, `undefined`.

Reply:
I’ve added a test to check if the function returns -Infinity for non-number values like true, null, and undefined.